modules/serverpod_auth/serverpod_auth_server/lib/src/business/password_hash.dart

```diff
@@ -0,0 +1,294 @@
+import 'dart:convert';
+import 'dart:math';
+import 'dart:typed_data';
+
+import 'package:crypto/crypto.dart';
+import 'package:pointycastle/key_derivators/api.dart';
+import 'package:pointycastle/key_derivators/argon2.dart';
+import 'package:serverpod_auth_server/module.dart';
+
+/// A class for handling password hashes.
+class PasswordHash {
+  static const int _saltLength = 16;
```

Comment:
16 bytes is the recommended salt length according to ietf, that is referenced from OWASP. https://www.ietf.org/archive/id/draft-ietf-kitten-password-storage-04.html#name-storage-2.

Since we generate the salt using only letters and numbers this should be 16 bytes.

Reply:
I'll add a link in the comment.